Mogakco/Sources/Presentation/Study/ViewController/StudyDetailViewController.swift

```diff
@@ -148,6 +140,7 @@ final class StudyDetailViewController: ViewController {
     override func bind() {
         let input = StudyDetailViewModel.Input(
             studyJoinButtonTapped: studyJoinButton.rx.tap.asObservable(),
+            participantCellTapped: participantsCollectionView.rx.itemSelected.asObservable(),
```

Comment:
```suggestion
            participantCellTapped: participantsCollectionView.rx.modelSelected(Patricipant.self).asObservable(),
```
요러면 인덱스로 안받아와도 되더라구요!

Reply:
혹시 ParticipantsCollectionView가 Participant 모델을 알고있을 수 있나요? ParticipantCell이 아닌 Participant 모델을 어떻게 알고 바로 넘겨주는지 잘 이해가 안됩니다..
그리고 지금은 그냥 delegate를 사용하지 않는데도 rx,itemSelected가 작동하지 않는데 이유를 알고 계신가요?